api/python/provisioner/commands/cluster_id.py

```diff
@@ -112,10 +115,12 @@ def _get_cluster_id(self, targets=ALL_MINIONS):
         res = None
         try:
             cluster_data = PillarGet().run('cluster', targets)
-            cluster_id = []
-
+            cluster_id= []
             for node in cluster_data:
-                if "cluster_id" in cluster_data[node]["cluster"]:
+                if cluster_data[node]["cluster"] is values.MISSED:
+                    print("cluster id is not avilable in pillar")
+                    res = None
```

Comment:
`res = None` is not required here as it is global. 
Only send out a `logger.debug()` on the scenario